core/src/main/java/com/orange/ouds/core/component/InteractionValuesIndication.kt

```diff
@@ -0,0 +1,361 @@
+    interactionState = interactionState,
+    toAnimatableColor = { it },
+    fromAnimatableColor = { it })
+
+@Composable
+internal fun rememberInteractionColor(
+    interactionState: InteractionState,
+    init: @Composable (InteractionState) -> Color
+) = rememberInteractionColor(
+    none = init(InteractionState.None),
+    focused = init(InteractionState.Focused),
+    hovered = init(InteractionState.Hovered),
+    pressed = init(InteractionState.Pressed),
+    interactionState = interactionState
+)
+
+@Composable
+internal fun rememberNullableInteractionColor(
+    none: Color?,
+    focused: Color?,
+    hovered: Color?,
+    pressed: Color?,
+    interactionState: InteractionState
+) = rememberInteractionValue(
+    none = none,
+    focused = focused,
+    hovered = hovered,
+    pressed = pressed,
+    interactionState = interactionState,
+    toAnimatableColor = { it.orElse { Color.Transparent } },
+    fromAnimatableColor = { it })
+
+@Composable
+internal fun rememberNullableInteractionColor(
+    interactionState: InteractionState,
+    init: @Composable (InteractionState) -> Color?
+) = rememberNullableInteractionColor(
+    none = init(InteractionState.None),
+    focused = init(InteractionState.Focused),
+    hovered = init(InteractionState.Hovered),
+    pressed = init(InteractionState.Pressed),
+    interactionState = interactionState
+)
+
+@Composable
+@JvmName("rememberAnimatableFloatInteractionValue")
+internal fun <T> rememberInteractionValue(
+    none: T,
+    focused: T,
+    hovered: T,
+    pressed: T,
+    interactionState: InteractionState,
+    toAnimatableFloat: (T) -> Float,
+    fromAnimatableFloat: (Float) -> T
+): InteractionValue<T, Float> = remember(none, focused, hovered, pressed, toAnimatableFloat, fromAnimatableFloat) {
+    AnimatableFloatInteractionValue(none, focused, hovered, pressed, toAnimatableFloat, fromAnimatableFloat)
+}.apply {
+    this.interactionState = interactionState
+}
+
+@Composable
+internal fun <T> rememberInteractionValue(
+    interactionState: InteractionState,
+    toAnimatableFloat: (T) -> Float,
+    fromAnimatableFloat: (Float) -> T,
+    init: @Composable (InteractionState) -> T
+) = rememberInteractionValue(
+    none = init(InteractionState.None),
+    focused = init(InteractionState.Focused),
+    hovered = init(InteractionState.Hovered),
+    pressed = init(InteractionState.Pressed),
+    interactionState = interactionState,
+    toAnimatableFloat = toAnimatableFloat,
+    fromAnimatableFloat = fromAnimatableFloat
+)
+
+@Composable
+internal fun rememberInteractionFloat(
+    none: Float,
+    focused: Float,
+    hovered: Float,
+    pressed: Float,
+    interactionState: InteractionState
+) = rememberInteractionValue(
+    none = none,
+    focused = focused,
+    hovered = hovered,
+    pressed = pressed,
+    interactionState = interactionState,
+    toAnimatableFloat = { it },
+    fromAnimatableFloat = { it })
+
+@Composable
+internal fun rememberInteractionFloat(
```

Comment:
This method is unused. If we decide to remove it, the previous one in this file becomes useless.